dev-server/ui-playground/components/ButtonPatterns.js

```diff
@@ -0,0 +1,157 @@
+import {
+  PatternPage,
+  Pattern,
+  PatternExamples,
+  PatternExample,
+} from '../shared/components/PatternPage';
+
+import {
+  IconButton,
+  LabeledButton,
+  LinkButton,
+} from '../../../src/shared/components/buttons';
+
+export default function ButtonPatterns() {
+  return (
+    <PatternPage title="Buttons">
+      <Pattern title="PublishControlButton">
+        <p>
+          Customizes <code>LabeledButton</code> styling to disable
+          <code>border-radius</code> on the right side. This makes the publish
```

Comment:
Ooof. This is going to be "fun" as some combination of my editor/prettier wants to hard-wrap long lines when they're within `<p>` elements.

Reply:
Update: This came about because of a combination of Prettier-imposed line breaks and manually-inserted ones. Prettier knows when it needs to insert a `{' '}` when imposing its own line breaks, so if I always fall back on Prettier and don't try to do my own thinking, this works OK.